src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -666,6 +668,36 @@ public static void setDistanceModelUpdateUrl(String url) {
         distanceModelUpdateUrl = url;
     }
 
+    /**
+     * Default class for rssi filter/calculation implementation
+     */
+    protected static Class rssiFilterImplClass = RunningAverageRssiFilter.class;
```

Comment:
This gives me a type error – IntelliJ or Java (?) is not able to do or allow that casting (from specific class to a type)

I get: Incompatible types.

 

Can you please check!?

 

Von: David G. Young [mailto:notifications@github.com] 
Gesendet: Mittwoch, 4. März 2015 14:01
An: AltBeacon/android-beacon-library
Cc: IgorZaus
Betreff: Re: [android-beacon-library] Changes for #113 and #114 (#128)

 

In src/main/java/org/altbeacon/beacon/BeaconManager.java <https://github.com/AltBeacon/android-beacon-library/pull/128#discussion_r25772294> :

> @@ -666,6 +668,36 @@ public static void setDistanceModelUpdateUrl(String url) {
>          distanceModelUpdateUrl = url;
>      }
>  
> +    /**
> +     * Default class for rssi filter/calculation implementation
> +     */
> +    protected static Class rssiFilterImplClass = RunningAverageRssiFilter.class;

Yes, I I just mean use RssiFilter instead of Class like this:

    protected static RssiFilter rssiFilterImplClass = RunningAverageRssiFilter.class;

—
Reply to this email directly or view it on GitHub <https://github.com/AltBeacon/android-beacon-library/pull/128/files#r25772294> .  <https://github.com/notifications/beacon/AKbDOcliTLEg1KRDUp-iM-ENBxSO8DLMks5nxvlxgaJpZM4DiTqq.gif>